docs/deployment/images.md

```diff
@@ -55,3 +55,31 @@ root@dokku:~# dokku tags:deploy node-js-app v0.9.0
        http://node-js-app.dokku.me
 
 ```
+
+## Add Image Directly from Docker Hub
+
+Alternatively you can pull an image from Docker Hub and deploy it using the tagging feature.
+
+1. Create dokku app as usual
+
+       ```
+       root@dokku:~# dokku apps:create test-app
+       ```
+
+2. Pull image from Docker Hub
+
+       ```
+       root@dokku:~# docker pull demo-repo/some-image:v12
+       ```
+
+3. Retag the image to match the created app
+
+       ```
+       root@dokku:~# docker tag demo-repo/some-image:v12 dokku/test-app:latest
```

Comment:
Sure, will double check.
On Wed, Apr 13, 2016 at 10:19 PM Jose Diaz-Gonzalez <
notifications@github.com> wrote:

> In docs/deployment/images.md
> <https://github.com/dokku/dokku/pull/2097#discussion_r59655770>:
>
> > +1. Create dokku app as usual
> > +
> > +       ```
> > +       root@dokku:~# dokku apps:create test-app
> > +       ```
> > +
> > +2. Pull image from Docker Hub
> > +
> > +       ```
> > +       root@dokku:~# docker pull demo-repo/some-image:v12
> > +       ```
> > +
> > +3. Retag the image to match the created app
> > +
> > +       ```
> > +       root@dokku:~# docker tag demo-repo/some-image:v12 dokku/test-app:latest
>
> Sounds like a bug, can you confirm with a set of logs?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/dokku/dokku/pull/2097/files/8d5991011485c7b5e7b74c54c0e90d0ac303fa4e#r59655770>
>